appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jmac/MessageSecurityConfigEventListenerImpl.java

```diff
@@ -77,7 +76,7 @@ public <T extends ConfigBeanProxy> NotProcessed handleDelete(T instance) {
     public <T extends ConfigBeanProxy> NotProcessed handleCreate(T instance) {
         LOG.log(Level.FINE, "MessageSecurityConfigEventListenerImpl - handleCreate called");
         if (instance instanceof MessageSecurityConfig) {
-            GFServerConfigProvider.loadConfigContext(service);
+            // GFServerConfigProvider.loadConfigContext(service);
```

Comment:
What about this? Remove or uncomment? Don't leave commented code in the codebase.